src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,7 @@
 *REMOVED*Microsoft.CodeAnalysis.TextDocument.Project.set -> void
 *REMOVED*Microsoft.CodeAnalysis.TextDocument.TextDocument() -> void
+Microsoft.CodeAnalysis.Host.IDocumentRefactoringService
+Microsoft.CodeAnalysis.Host.IDocumentRefactoringService.UpdateAfterInfoChangeAsync(Microsoft.CodeAnalysis.Document current, Microsoft.CodeAnalysis.Document previous, System.Threading.CancellationToken cancellation = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<Microsoft.CodeAnalysis.Solution>
```

Comment:
Let's take this through API design before merging. I feel good about it, but we should still discuss.